contrib/containerized-pbench/README.md

```diff
@@ -1,81 +1,118 @@
-# containerized-pbench
-
-## Openshift
-
-We can either build an image using the following commands or pull the image from dockerhub
-
-Build the image from source like
+## Containerized pbench
+This directory contains OPenShift templates, dockerfiles needed to build and run containerized pbench.
 
+### Prepare the nodes you want to monitor
+The nodes need to have pbench-agent, collectd images on it. The qcow images generated by the image provisioner already has the images baked in. In case the nodes doesn't have the images, please pull them from dockerhub:
```

Comment:
"already has the images baked in" --> "already have the necessary images baked in" perhaps. Also "the nodes doesn't have the images" --> "the nodes do not have the images"